### PR TITLE
NotificationStack component

### PR DESCRIPTION
**Pull request by Lindsay89**
Notification system

## Description
The NotificationsStack is a controlled component that required its state to be provided from the parent, meaning that an array of notifications and an onChange callback function should be provided. The notifications array is an array of objects: each object represent one notification, and it must have an id and a content property. The id is very important for the notification removal.

## Motivation and Context
The NotificationsStack component is useful to show notifications over the page to alert the user about something.

### CODE REVIEW

**Comment by antonioru** [comment]
@Lindsay89 

just a tiny missing: types support

File: src/components/Layout/NotificationsStack/README.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+};
+
+<> 
+  <UncontrolledNotification />
+</>
+```
+
+
+### Type
+The `type` prop is useful to let the user understand quickly the notification kind.
```
Comment (antonioru) [inline src/components/Layout/NotificationsStack/README.md]: Why is "type" and not "color"?

Comment (Lindsay89) [reply src/components/Layout/NotificationsStack/README.md]: Type is referring to the notification type: success notification, warning notification and so on...

File: @types/NotificationsStack.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/**
+ * Declares the NotificationsStack functional component
+ */
+
+declare const NotificationsStack: FunctionComponent<NotificationsStackProps>
+  & { Notification: FunctionComponent<NotificationProp> }
+  & { NotificationLeftRender: FunctionComponent<NotificationLeftRenderProp> }
+  & { NotificationContent: FunctionComponent<NotificationContentProp> }
+  & { NotificationRightRender: FunctionComponent<NotificationRightRenderProp> };
```
Comment (antonioru) [inline @types/NotificationsStack.ts]: Why are we declaring all the components as part of the NotificationsStack type?

File: CHANGELOG.md [hunk 3]
```diff
@@ -975,4 +975,12 @@ disable `Hot` and enable `live reloading` (it refresh when something changes)
 
 ### Fixing
 
-- Added ie support to the FloatingComponent by editing its util functions
\ No newline at end of file
+- Added ie support to the FloatingComponent by editing its util functions
+
+
+## [0.54.2] - 2020-03-19
```
Comment (antonioru) [inline CHANGELOG.md]: The version is 0.55.0

**Pull request merged**